benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -14820,8 +14820,8 @@ exports[`Render Summary for multi-table-only-columns.diff.html 1`] = `
        >
         <div
          className="cell"
-         dangerouslySetInnerHTML={".036"}
-         title="avg: 0.012, max: 0.015, median: 0.015, min: 0.006, stdev: 0.00424"
+         dangerouslySetInnerHTML={".036&#x2007;"}
+         title="avg: 0.0120, max: 0.015, median: 0.015, min: 0.006, stdev: 0.00424"
```

Comment:
As `avg` is calculated, should we fill missing significant digits with `0` here?

Reply:
It would be better to not do this.